lib/ApiClient.js

```diff
@@ -95,6 +95,12 @@ class ApiClient extends AxiosClient {
     return this.put(path, data);
   }
 
+  deleteDrop(code) {
+    const path = `drops/${code}`;
```

Comment:
Fun fact, it may be used also as: `DELETE /drops/foo,bar,baz` and then it will delete 3 drops `foo`, `bar` and `baz`. We can leave it as it is for now tho :)